groups/templates/groups/discussion_thread_base.html

```diff
@@ -18,9 +18,11 @@
         {% for comment in comments %}
             <li>
                 <ul>
+                    <li>
+                        <a name="{{ comment.get_pagejump_anchor }}"></a>
```

Comment:
As the ul seems to be the boundary of the comment, it's probably worth adding `name=` to that instead, and dropping the empty link.